dotCMS/src/main/resources/experiment/js/init_script.js

```diff
@@ -4,14 +4,41 @@ function setJitsuExperimentData (experimentData) {
         experiments: experimentData.experiments.map((experiment) => ({
                 experiment: experiment.id,
                 variant: experiment.variant.name,
-                lookBackWindow: experiment.lookBackWindow
+                lookBackWindow: experiment.lookBackWindow.value
             })
         )
     };
 
     jitsu('set', experimentsShortData);
 }
 
+window.addEventListener("experiment_loaded", function (event) {
+    let experimentData = event.detail;
+    setJitsuExperimentData(experimentData);
+
+    if (!window.location.href.includes("redirect=true")) {
+        for (let i = 0; i < experimentData.experiments.length; i++) {
+            let pageUrl = experimentData.experiments[i].pageUrl;
```

Comment:
Just a nit-pick if variables as `pageUrl`, `alternativePageUrl` and `url` are not changed at all, it's better to use `const`.

Reply:
done https://github.com/dotCMS/core/pull/24357/commits/00cea76efd0d901370aa6090e2efb4731a09e947#diff-9417bef73d32b061e75306b404886f1b00b867e22696a5ba0b0717473190c922R21